src/array/cuda/disjoint_union.cu

```diff
@@ -84,7 +84,7 @@ void _Merge(IdType** arrs, IdType* prefix, IdType* offset, IdType* out,
 
   device->CopyDataFromTo(
       arrs, 0, arrs_dev, 0, sizeof(IdType*)*n_arrs,
-      DGLContext{kDLCPU, 0}, ctx, dtype, 0);
+      DGLContext{kDLCPU, 0}, ctx, dtype, stream);
```

Comment:
Same. And probably remove `cudaStream_t stream` from arguments.

Reply:
Sure.